python/lsst/ts/vent/controller/controller.py

```diff
@@ -209,11 +245,35 @@ async def vfd_fault_reset(self) -> None:
             If a communications error occurs.
         """
 
+        assert self.connected
         for address, value in vf_drive.FAULT_RESET_SEQUENCE:
             await self.vfd_client.write_register(
                 slave=self.cfg.slave, address=address, value=value
             )
 
+    async def get_drive_state(self) -> FanDriveState:
+        """Returns the current fan drive state based on the contents
+        of the IPAE register.
+
+        Raises
+        ------
+        ModbusException
+            If a communications error occurs.
+        """
+
+        assert self.connected
+        ipae = (
```

Comment:
what doesn't `ipae` means? can you expand the acronym?

Reply:
It's called the IPAE register in the manual for the ATV320. They don't say what IPAE is supposed to mean or stand for. I've added some clarification in the comments.